sormas-e2e-tests/src/main/java/org/sormas/e2etests/services/api/EventApiService.java

```diff
@@ -35,7 +35,7 @@ public EventApiService() {}
   public Event buildGeneratedEvent() {
     return Event.builder()
         .uuid(UUID.randomUUID().toString())
-        .disease(DiseasesValues.getRandomDiseaseName())
+        .disease(DiseasesValues.CORONAVIRUS.getDiseaseName())
```

Comment:
I checked before changing it If it Will affect any Of the tests, but this field was added by Michal And tests he wrote are passing with disease set to covid. In this test I had to create multiple casea with the same disease as event that is why I decided to change it from random to covid disease